src/routes/Filters.tsx

```diff
@@ -0,0 +1,110 @@
+import { FilterRemoveIcon } from '@primer/octicons-react';
+import { type FC, useContext } from 'react';
+import { Header } from '../components/Header';
+import { Checkbox } from '../components/fields/Checkbox';
+import { AppContext, defaultSettings } from '../context/App';
+import { BUTTON_CLASS_NAME } from '../styles/gitify';
+import { Size } from '../types';
+import type { Reason } from '../typesGitHub';
+import { FORMATTED_REASONS, formatReason } from '../utils/reason';
+
+export const FiltersRoute: FC = () => {
+  const { settings, updateSetting } = useContext(AppContext);
+
+  const updateReasonFilter = (reason: Reason, checked: boolean) => {
+    let reasons: Reason[] = settings.filterReasons;
+
+    if (checked) {
+      reasons.push(reason);
+    } else {
+      reasons = reasons.filter((r) => r !== reason);
+    }
+
+    updateSetting('filterReasons', reasons);
+  };
+
+  const shouldShowReason = (reason: Reason) => {
+    return settings.filterReasons.includes(reason);
+  };
+
+  const resetToDefaultFilters = () => {
+    updateSetting('filterReasons', defaultSettings.filterReasons);
+    updateSetting('showBots', defaultSettings.showBots);
+  };
+
+  return (
+    <div className="flex h-screen flex-col" data-testid="filters">
+      <Header fetchOnBack={true}>Filters</Header>
+      <div className="flex-grow overflow-x-auto px-8">
+        <fieldset className="mb-3">
+          <legend id="notifications" className="mb-1 mt-2 font-semibold">
+            Users
+          </legend>
+          <Checkbox
+            name="showBots"
+            label="Show notifications from Bot accounts"
```

Comment:
What if we change this into a "Hide notifications from Bot Accounts" instead? This way all filters would be disabled by default.

Reply:
Good suggestion. I'll make that change shortly

Reply:
Renamed the setting and updated labels as suggested